src/cheri/insns/cmv_16bit.adoc

```diff
@@ -15,8 +15,6 @@ Expansion::
 Suggested assembly syntax::
 `mv cd, cs2`
 
-NOTE: This encoding is shared with integer `c.mv`.
```

Comment:
I think it's still useful to say that we remap this encoding? There is no integer move in compressed opcode space.